subworkflows/nf-core/fastq_qc_trim_filter_setstrandedness/main.nf

```diff
@@ -114,6 +116,23 @@ workflow FASTQ_QC_TRIM_FILTER_SETSTRANDEDNESS {
     ch_trim_read_count = Channel.empty()
     ch_multiqc_files   = Channel.empty()
 
+    ch_reads
+        .map {
+            meta, fastqs ->
+            return [meta, fastqs.flatten()]
+        }
+        .set { ch_fastq_lint }
+    
+    //
+    // MODULE: Lint FastQ files
+    //
+    if (!skip_linting) {
+        FQ_LINT (
+            ch_fastq_lint
+        )
+        ch_versions = ch_versions.mix(FQ_LINT.out.versions.first())
+    }
```

Comment:
```suggestion    
    //
    // MODULE: Lint FastQ files
    //
    if (!skip_linting) {
        FQ_LINT (
            ch_fastq_lint.map{ meta, fastqs -> [meta, fastqs.flatten()] }
        )
        ch_versions = ch_versions.mix(FQ_LINT.out.versions.first())
    }
```

(but again, this needs PR'ing to the modules repo first)

Reply:
Also, the linting outputs should be added as a subworkflow output

Reply:
Thanks so much for reviewing @pinin4fjords ! I opened a PR in the modules repo here: https://github.com/nf-core/modules/pull/7000